mdoc/Resources/monodoc-ecma.xsd

```diff
@@ -485,6 +501,7 @@ add masterdoc support?
   <xs:element name="Member">
     <xs:complexType>
       <xs:sequence>
+        <xs:element ref="Meta" minOccurs="0" maxOccurs="1" />
```

Comment:
should this be "Metadata" instead of "Meta"?

Reply:
Yes indeed! great catch ... thanks for the review :)